build-tests/install-test-workspace/workspace/typescript-v3-test/package.json

```diff
@@ -11,7 +11,7 @@
   "devDependencies": {
```

Comment:
This package name is different from the folder it lives in. Is this supposed to be a project that uses TS v3? @octogonz?